CRM/Member/BAO/Query.php

```diff
@@ -312,6 +312,18 @@ public static function whereClauseSingle(&$values, &$query) {
         }
         $query->_tables['civicrm_membership'] = $query->_whereTables['civicrm_membership'] = 1;
         return;
+
+      case 'member_override':
```

Comment:
And I agree field name should be membership_is_override since membership.get.api expects the same name.